<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>239</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>-4</x>
      <y>0</y>
      <width>251</width>
      <height>421</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="widget">
     <attribute name="title">
      <string>Control</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>10</y>
        <width>109</width>
        <height>71</height>
       </rect>
      </property>
      <property name="title">
       <string>Управление</string>
      </property>
      <widget class="QCheckBox" name="light1">
       <property name="geometry">
        <rect>
         <x>13</x>
         <y>23</y>
         <width>57</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Свет 1</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="fan">
       <property name="geometry">
        <rect>
         <x>13</x>
         <y>46</y>
         <width>83</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Вентилятор</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>97</width>
        <height>111</height>
       </rect>
      </property>
      <property name="title">
       <string>Serial</string>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>81</width>
         <height>84</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QComboBox" name="ports"/>
        </item>
        <item>
         <widget class="QPushButton" name="connectB">
          <property name="text">
           <string>Connect</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="disconnectB">
          <property name="text">
           <string>Disconnect</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QLabel" name="microphoneImage">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>221</width>
        <height>221</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../Downloads/imgonline-com-ua-Resize-ufyv4e2dQsRZumK.jpg</pixmap>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Sencors</string>
     </attribute>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
